.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+      {
+        "name": "Minitest - current file",
+        "type": "ruby_lsp",
+        "request": "launch",
+        "program": "rake test TEST=${file}"
+      }
+    ]
+  }
```

Comment:
please lets avoid any unnecessary IDE related staff out

Reply:
Good point, I added these only before finding out that adding `rake` might make sense, and now `ruby-lsp` runs tests just fine. Removing!